installer/helpers.sh

```diff
@@ -1,6 +1,23 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+OPNAMESPACE="porter-operator-system"
```

Comment:
This works but I wanted you to know about a porter feature in case that affects how you'd like to implement this. 

The parameter operatorNamespace is always populated, it will have the default value if not specified by the user, and a corresponding environment variable, `OPERATORNAMESPACE` set. There isn't a need to define OPNAMESPACE, pass the parameter as an argument to each helper function, and initialize OPNAMESPACE in each function. You can just use OPERATORNAMESPACE.